skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/LinuxRedrawer.kt

```diff
@@ -37,10 +30,18 @@ internal class LinuxRedrawer(
         frameDispatcher.scheduleFrame()
     }
 
-    private suspend fun update(nanoTime: Long) {
-        withContext(job) {
-            layer.update(nanoTime)
-        }
+    override fun redrawImmediately() = layer.lockDrawingSurface {
+        check(!isDisposed)
+        update(System.nanoTime())
+        it.makeCurrent(context)
+        draw()
+        it.setSwapInterval(0)
+        it.swapBuffers()
+        OpenGLApi.instance.glFinish()
```

Comment:
It doesn’t look right to add it here, can we make backend-specific sync code.

Reply:
LinuxRedrawer/WindowsRedrawer/MacOsRedrawer are bound completely to OpenGL.

We can rename them to LinuxOpenGLRedrawer/WindowsOpenGLRedrawer/MacOsOpenGLRedrawer

Reply:
Extract backend-independent code we should only after we implement another hardware backends.
Before that we don't know exactly what is backend-independent :).

Reply:
Maybe merge after https://github.com/JetBrains/skiko/pull/56 and put move code to proper place?

Reply:
Event after #56 this is the proper place. #56 only adds raster render and platform for other renderings.

All functions (createContext, makeCurrent, swapBuffers) are OpenGL specific, if I am not mistaken.

Reply:
OK, then probably slight rename of classes, or eventual splitting redrawer into backend independent and dependent parts looks reasonable. Anyway, if it works OK with software rasterizer - I'm fine with it.

Reply:
Ok, will rename and merge after #56

Reply:
Renamed